test/e2e/specs/publishing.test.js

```diff
@@ -76,4 +77,30 @@ describe( 'Publishing', () => {
 			} );
 		} );
 	} );
+
+	[ 'post', 'page' ].forEach( ( postType ) => {
+		let werePrePublishChecksEnabled;
+		describe( `a ${ postType } in small viewports`, () => {
+			beforeEach( async () => {
+				await newPost( postType );
+				werePrePublishChecksEnabled = await arePrePublishChecksEnabled();
+				if ( werePrePublishChecksEnabled ) {
+					await disablePrePublishChecks();
+				}
+				await setViewport( 'small' );
+			} );
+
+			afterEach( async () => {
```

Comment:
Why have the `beforeEach` and `afterEach` callbacks if we only have one test?

Reply:
Because I was hoping to add more tests, but I've failed to. I would still leave them as they are, in case I can revisit before this is merged.